docs/how-to/slice-a-package.md

```diff
@@ -0,0 +1,626 @@
+  has a `core` slice providing a very minimal `python3` runtime, but also a
+  `standard` slice with the additional libraries on top of `core`.
+    ```{tip}
+    In this case, the most common is to create:
+    - a `core` slice which contains the very minimal, core set of files
+      supporting a minimal yet functional operation of the application,
+    - a `standard` slice which contains a standard set of files supporting the
+      full operation of the application,
+    - and other slices which are scoped and named after the functionality they
+      provide (e.g. see the [`libpython3.12-stdlib` slices](https://github.com/canonical/chisel-releases/blob/ubuntu-24.10/slices/libpython3.12-stdlib.yaml)).
+    ```
+```
+
+6. [ ] **Choose the design approach that aligns best with your package**
+
+    Given the {ref}`above package inspection <create_sdf_inspect_pkg>`, the
+    `openssl` package can be sliced by *type of content*:
+     - a `bins` slice for the `/usr/bin/openssl` binary,
+       - note that although `c_rehash` is also under `/usr/bin/`, it is
+       actually a Perl script,
+     - a `config` slice for the {ref}`configuration files <create_sdf_inspect_pkg_conffiles>`,
+     - a `data` slice for the package's certificates files, and
+     - a `copyright` slice for the copyright file.
+
+```{important}
+**Every slice** must install the package's copyright file(s).
+
+It is therefore recommended to create a `copyright` slice for every package, that
+other slices can depend on.
+```
+
+7. [ ] **Prepare to write the slice definitions**
+
+    ```{note}
+    There is one Slice Definitions File per package!
+    ```
+
+    Slice definitions live in the `slices/` directory of the
+    {{chisel_releases_repo}}. Fork and clone this repo, creating a new
+    branch for the packages you are slicing.
+
+8. [ ] **Create the YAML Slice Definitions File**
+
+   Create a new file in the `slices/` directory, named after the package you
+   are slicing. For example, `openssl.yaml`.
+
+9. [ ] **Specify `package` name**
+
+   The first thing to do is to write the package name in the
+   {ref}`slice_definitions_format_package` field. For example, open
+   `slices/openssl.yaml` and write:
+
+   ```yaml
+   package: openssl
+   ```
+
+10. [ ] **Write the `copyright` slice**
+
+    Write the copyright slice first and pin it as an {ref}`essential for every
+    other package slice <slice_definitions_format_essential>`.
+
+    ```yaml
+    essential:
+      - openssl_copyright
+
+    slices:
+      copyright:
+        contents:
+          # This path is taken from the package contents inspected above.
+          /usr/share/doc/openssl/copyright:
+          # Notice that this path is in fact a symlink to a copyright file in
+          # the libssl3t64 package (which is a dependency of openssl).
+          # So we need to add that package's copyright slice as a dependency.
+        essential:
+          - libssl3t64_copyright
```

Comment:
I think we typically put the `essential` first and then the `contents`.

Reply:
changes